lmdeploy/turbomind/deploy/source_model/llama.py

```diff
@@ -236,6 +239,10 @@ def model_info(self):
                     else llama3_scaling_type
                 if scaling_type == 'dynamic':
                     use_dynamic_ntk = 1
+                attention_factor = model_arg['rope_scaling'].get(
+                    'attention_factor', None)
```

Comment:
https://github.com/huggingface/transformers/blob/f2c388e3f946862f657acc1e21b272ec946fc66c/src/transformers/modeling_rope_utils.py#L198
```
attention_factor = config.rope_scaling.get("attention_factor")
    if attention_factor is None:
        attention_factor = 0.1 * math.log(factor) + 1.0
```

Reply:
The [previous code](https://github.com/InternLM/lmdeploy/blob/152d4be19bf271a8a47ec5c54c8a3da7a3dc651c/src/turbomind/models/llama/unified_attention_layer.cc#L333-L338) had this logic. And add this when converting the model as well.